app/assets/javascripts/families.js

```diff
@@ -0,0 +1,17 @@
+function showActiveFamilies(){
+  document.getElementById("active_families").style.display = "block";
+  document.getElementById("not_active_families").style.display = "none";
+  document.getElementById("all_families").style.display = "none";
+}
+
+function showNotActiveFamilies(){
+  document.getElementById("not_active_families").style.display = "block";
+  document.getElementById("active_families").style.display = "none";
+  document.getElementById("all_families").style.display = "none";
+}
+
+function showAllFamilies(){
+  document.getElementById("all_families").style.display = "block";
+  document.getElementById("not_active_families").style.display = "none";
+  document.getElementById("active_families").style.display = "none";
+}
```

Comment:
A suggestion to rewrite this to use a function to hide all, and then just turn one on.
Also, to use css classes for the hiding and showing by adding/removing a class. For example:

```
const buttons = document.querySelectorAll(".button-families");

function showFamilies(id_name) {
  buttons.forEach(button => button.classList.remove("active"));
  document.querySelector(id_name).classList.add("active");
}
```

Reply:
Actually, you can build it to be generic enough, so it will also work with Trips. Happy to help with this if you wish